.github/readmes/javascript/_evolving-the-app-graphql-sdl.md

```diff
@@ -3,9 +3,9 @@
 Evolving the application typically requires two steps:
 
 1. Migrate your database using Prisma Migrate
-1. Update your application code
+2. Update your application code
```

Comment:
```suggestion
1. Update your application code
```

Reply:
In numbered lists in Markdown, I prefer writing them all out as `1.` because:

1. that way
1. it will be easier
1. to reorder them or insert new list items later
1. and the cool thing is
1. markdown still renders them as incrementing numbers

![image](https://user-images.githubusercontent.com/4058327/110650034-77dffc00-81ba-11eb-968c-82353cf038e7.png)